packages/react/src/components/ComboBox/ComboBox-test.js

```diff
@@ -135,7 +136,16 @@ describe('ComboBox', () => {
     it('should not let the user expand the menu', () => {
       const wrapper = mount(<ComboBox {...mockProps} disabled={true} />);
       openMenu(wrapper);
-      expect(findMenuNode(wrapper).length).toBe(0);
+      // expect(findMenuNode(wrapper).length).toBe(0);o
```

Comment:
Do you want to remove the commented-out lines here...?

Reply:
@asudoh i'm currently looking at this line to make sure i choose the right testing statement, will push update soon!